pkg/cmd/server/api/types.go

```diff
@@ -1004,6 +1004,9 @@ type LDAPQuery struct {
 
 	// Filter is a valid LDAP search filter that retrieves all relevant entries from the LDAP server with the base DN
 	Filter string
+
+	// PageSize is the maxmimum preferred page size, measured in LDAP entries
+	PageSize int
```

Comment:
Do we really think that someone will set this?   Why not choose a number between 100 and 1000 and call it a day?